romancal/resample/tests/test_resample.py

```diff
@@ -620,3 +620,72 @@ def test_resampledata_do_drizzle_default_single_exposure_weight_array(
 
     assert np.any(output_models_many_to_one[0].weight > 0)
     assert np.any(output_models_many_to_many[0].weight > 0)
+
+
+def test_populate_mosaic_basic(exposure_1):
```

Comment:
This looks good to me, but I think I would either simplify the tests or give it some more challenging tests?  If I understand correctly, you've got the code for handling cases like multiple surveys here, but you're only testing a single exposure where most of the test code isn't needed?

Reply:
Done.